File: java/terreno.java
import java.util.Locale;
import java.util.Scanner;

public class terreno {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double largura, comprimento, valor, area, preco;

	    System.out.print("Digite a largura do terreno: ");
	    largura = sc.nextDouble();

	    System.out.print("Digite o comprimento do terreno: ");
	    comprimento = sc.nextDouble();

	    System.out.print("Digite o valor do metro quadrado: ");
	    valor = sc.nextDouble();

	    area = largura * comprimento;

	    System.out.printf("Area do terreno = %.2f\n", area);

	    preco = area * valor;

	    System.out.printf("Preco do terreno = %.2f\n", preco);

		sc.close();
	}
}

File: python/terreno.py
largura: float; comprimento: float; valor: float; area: float; preco: float;

largura = float(input("Digite a largura do terreno: "))
comprimento = float(input("Digite o comprimento do terreno: "))
valor = float(input("Digite o valor do metro quadrado: "))

area = largura * comprimento

print(f"Area do terreno = {area:.2f}")

preco = area * valor

print(f"Preco do terreno = {preco:.2f}\n")
